basics/assets/the-fil-token.md

```diff
@@ -10,21 +10,19 @@ description: >-
 
 FIL plays a vital role in incentivizing users to participate in the Filecoin network and ensuring its smooth operation. Here are some ways in which FIL is used on the Filecoin network:
 
-### Storage payments
+### Network payments
 
-When a user wants to store data on the Filecoin network, they pay in FIL to the storage providers who offer their storage space. The payment is made in advance for a certain amount of time that the data will be stored on the network. The storage providers are then rewarded with FIL for providing their storage space and performing other useful tasks on the network.
+When a user wants to store data on the Filecoin network, they pay in FIL to the storage providers who offer their storage space. The payment is made in advance, for a certain amount of time that the data will be stored on the network.
```

Comment:
payment doesn't go to SP in advance, it's held by the storage market actor, "vesting" together with the age of the sector

this is all in the [spec](https://spec.filecoin.io/)